### ISSUE TITLE
Original folders view instead of artist listing

### ISSUE THREAD

**Issue by primg** [clementine-player/Android-Remote#138]
By only displaying the artist with their corresponding tracks, it split up all the compilations or mixes album.
Could be great to have the listing of the original folder view from the library.

**Comment by amuttsch** [comment]
With the new version it is possible to group the library by Album-Artist and more like in Clementine. A listing of the original folders in impractical imho, because it might cause more navigation than necessary.

**Comment by postags** [comment]
I disagree, often large collections are divided into folder structures to make browsing less arduous. 

I have about ~50k songs that break down into something like 4000 albums. Browsing through that many albums to find a specific one is tiresome, especially since I'm not usually looking for a particular album, or even a particular artist, but something in a particular genre. I understand that the app has a Genre-Artist-Album browsing structure but it (1) both consistently stutters while navigating a list of 10 genres, and (2) requires me to develop a whole additional ingestion routine to make sure that the genre tags that are actually useful to this specific app.... which is extraneous, since I've already filed it in its appropriate place in the folder hierarchy. 

This doesn't even touch what a nightmare it would be to navigate an album list of prolific artists who have hundreds of albums, or hundreds of live shows, or for collections of songs by producers who have credits extending across hundreds of albums, for which a user might want the original tags intact but has actually grouped into a series of subfolders, or really any other idiosyncratic need that makes folders make more sense. Tagging structure can only do so much when it comes to these use-cases.

Directory structure navigation is actually the primary reason I use Clementine in the first place, so having it in the app would be a huge boon to me and others like me.
